xc/common/cmake/arch_define.cmake

```diff
@@ -75,7 +75,10 @@ function(ADD_XC_ARCH_DEFINE)
       --incremental_reroute_delay_ripup off \
       --base_cost_type delay_normalized_length_bounded \
       --bb_factor 10 \
-      --initial_pres_fac 4.0 \
+      --acc_fac 0.7 \
```

Comment:
Let's add a comment pointing to documentation on how these parameters were chosen.  E.g. we swept the following circuits, over the following parameters, etc.

Reply:
Should this go somewhere in `docs/`, or as comments in this file?

Reply:
A comment pointing to more details docs is probably good here.

Reply:
Agreed, comment that is a pointer to a doc is probably good here.

Reply:
I've added documentation inline, and a link to the detailed report.